org.shadered.SHADERed.yaml

```diff
@@ -0,0 +1,79 @@
+app-id: org.shadered.SHADERed
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+default-branch: stable
```

Comment:
```suggestion
```
it's the default already

Reply:
The default branch is `master`, flathub just overrides it.

Reply:
@TingPing Should I keep `default-branch: stable` then?